Calibration/IsolatedParticles/interface/CaloConstants.h

```diff
@@ -0,0 +1,25 @@
+#ifndef CalibrationIsolatedParticlesCaloConstants_h
+#define CalibrationIsolatedParticlesCaloConstants_h
+
+#include <cmath>
+
+namespace spr{
+
+  static const double deltaEta = 0.087; //Tower size
+  static const double zFrontEE = 319.2; //Front of EE
+  static const double rFrontEB = 129.4; //Front of EB
+  static const double zFrontES = 303.2; //Front of ES
+  static const double etaBEEcal= 1.479; //Transition eta between EB and EE
+  static const double zFrontHE = 402.7; //Front of HE
+  static const double rFrontHB = 180.7; //Front of HB
+  static const double etaBEHcal= 1.392; //Transition eta between HB and HE
+  static const double zBackHE  = 549.3; //Back of HE
+  static const double rBackHB  = 288.8; //Back of HB
+  static const double rFrontHO = 384.8; //Front of HO
+  static const double zFrontHF = 1115.; //Front of HF
+  static const double zFrontTE = 110.0; //Front of Tracker endcap
+  static const double zBackTE  = 290.0; //Back of Tracker endcap
+  static const double rBackTB  = 109.0; //Back of Tracker barrel
+  static const double etaBETrak= 1.705; //Transition eta between barrel-endcap of Tracker
+}
```

Comment:
It is not very easy to do that. That is why I have put in those constants by hand
________________________________
From: Marco Musich [notifications@github.com]
Sent: 26 January 2016 18:04
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-alca32 Remove some of the magic numbers and update the calibration macro (#12859)


In Calibration/IsolatedParticles/interface/CaloConstants.h<https://github.com/cms-sw/cmssw/pull/12859#discussion_r50865067>:

> +  static const double zFrontEE = 319.2; //Front of EE
> +  static const double rFrontEB = 129.4; //Front of EB
> +  static const double zFrontES = 303.2; //Front of ES
> +  static const double etaBEEcal= 1.479; //Transition eta between EB and EE
> +  static const double zFrontHE = 402.7; //Front of HE
> +  static const double rFrontHB = 180.7; //Front of HB
> +  static const double etaBEHcal= 1.392; //Transition eta between HB and HE
> +  static const double zBackHE  = 549.3; //Back of HE
> +  static const double rBackHB  = 288.8; //Back of HB
> +  static const double rFrontHO = 384.8; //Front of HO
> +  static const double zFrontHF = 1115.; //Front of HF
> +  static const double zFrontTE = 110.0; //Front of Tracker endcap
> +  static const double zBackTE  = 290.0; //Back of Tracker endcap
> +  static const double rBackTB  = 109.0; //Back of Tracker barrel
> +  static const double etaBETrak= 1.705; //Transition eta between barrel-endcap of Tracker
> +}


@bsunanda<https://github.com/bsunanda> isn't there any other way to get this numbers (e.g. from the Geometry?)

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/12859/files#r50865067>.